linux_sql/scripts/psql_docker.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# arguments
+job=$1
+db_username=$2
+db_password=$3
+
+# start docker if not already running
+systemctl status docker || systemctl start docker
+
+# check number of arguments: at least 1
+if [ "$#" -eq 0 ]; then
+  echo "Illegal number of parameters"
+  exit 1
+fi
+
+# determine the command passed into script
+case $job in
+  create)
+    # print error message if
+    if [ "$#" -lt 3 ]; then
+      echo "Illegal number of parameters for 'create'"
+      exit 1
+    fi
+    # raise error if container already exists
+    container_num=$(($(docker container ls -a -f name=jrvs-psql | wc -l)-1))
```

Comment:
do not hard code container name (e.g. `jrvs-psql`). you should define it after line 6